test/framework/src/main/java/org/elasticsearch/index/engine/EngineTestCase.java

```diff
@@ -895,9 +895,9 @@ protected EngineConfig noOpConfig(IndexSettings indexSettings, Store store, Path
         return config(indexSettings, store, translogPath, newMergePolicy(), null, null, globalCheckpointSupplier);
     }
 
-    protected static final BytesReference B_1 = new BytesArray(new byte[] { 1 });
-    protected static final BytesReference B_2 = new BytesArray(new byte[] { 2 });
-    protected static final BytesReference B_3 = new BytesArray(new byte[] { 3 });
+    protected static final BytesReference B_1 = bytesArray("{\"b\":1}");
```

Comment:
Also out of curiosity, why this change?

Reply:
I've added some assertions in `Translog.Index#invariant()` to assert the XContentType stored in the translog is known and correct, so I had to adapt most tests even if those tests do not strictly use the source.